.travis.yml

```diff
@@ -50,3 +50,6 @@ services:
   - rabbitmq
 addons:
   postgresql: "9.4"
+  apt:
+    packages:
+    - postgresql-9.4
```

Comment:
I confirmed that travis uses 9.4.0.

https://github.com/mtsmfm/railstest/pull/5/commits/f23119f4b9499ec30565981198110124c8a2e1d8

If we have this hack:

https://travis-ci.org/mtsmfm/railstest/jobs/176084769/config

```
"addons": {
    "postgresql": "9.4",
    "apt": {
      "packages": [
        "postgresql-9.4"
      ]
    }
  },
```

postgres will >= 9.4.2 and we can use alter constraint:

https://travis-ci.org/mtsmfm/railstest/jobs/176084769#L564

> run PostgreSQLReferentialIntegrityWithAlterConstraintTest

But if we don't have:

https://travis-ci.org/mtsmfm/railstest/jobs/176084770/config

```
"addons": {
    "postgresql": "9.4"
  },
```

https://travis-ci.org/mtsmfm/railstest/jobs/176084770

Enabled fall back:

https://travis-ci.org/mtsmfm/railstest/jobs/176084770#L529